spec/helper.rb

```diff
@@ -10,12 +10,10 @@
 require 'base64'
 require 'jwt'
 # latest version of pry-byebug is not compatible with Ruby 3.2.0
-if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('3.2.0')
-  require 'pry-byebug'
-end
+require 'pry-byebug' if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('3.2.0')
```

Comment:
We should check if `pry-byebug` still has this problem and remove it when it doesn't

Reply:
Issue created: https://github.com/octokit/octokit.rb/issues/1439